R/utils-assert.R

```diff
@@ -77,8 +78,8 @@ assert_relatives <- function(x, name = deparse(substitute(x))) {
 
 
 assert_logical <- function(x, name = deparse(substitute(x))) {
-  if (!(x %in% 0:1 || is.logical(x))) {
-    stop(sprintf("'%s' must be logical or in {0, 1}", name), call. = FALSE)
+  if (!all(x %in% 0:1 | is.logical(x))) {
```

Comment:
is.logical applies to whole vector so should use `||`; how about:

```suggestion
  if (!(all(x %in% 0:1) || is.logical(x))) {
```